### PR TITLE
Add setting for loadbalancer IPs

### PR DESCRIPTION
**Pull request by ben221199**
This PR adds the possibility to set loadbalancer IPs in `zulip.conf` using Docker container environment parameters.

### CODE REVIEW

**Comment by ben221199** [comment]
@timabbott Could this be merged?

**Comment by ben221199** [comment]
@andersk @alexmv

File: docker-compose.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ services:
     volumes:
       - "/opt/docker/zulip/redis:/data:rw"
   zulip:
-    image: "zulip/docker-zulip:5.2-0"
+    image: "zulip/docker-zulip:5.2-1"
```
Comment (timabbott) [inline docker-compose.yml:50]: Remove this part -- we do version updates in separate release commits.

File: README.md [hunk 2]
```diff
@@ -209,6 +209,10 @@ putting it in `/opt/docker/zulip/zulip/certs/` (by default, the
 `zulip` container startup script will generate a self-signed certificate and
 install it in that directory).
 
+**Loadbalancer**. To tell Zulip it's behind a loadbalancer, you can set
+`LOADBALANCER_IPS` to a comma-separated list of IPs. This will tell Zulip
+to pass the real IP of the client instead of the IP of the loadbalancer itself.
```
Comment (timabbott) [inline README.md:214]: This should reference the setting for and link to the ReadTheDocs discussion of the `[loadbalancer]` setting.

File: README.md [hunk 3]
```diff
@@ -209,6 +209,10 @@ putting it in `/opt/docker/zulip/zulip/certs/` (by default, the
 `zulip` container startup script will generate a self-signed certificate and
 install it in that directory).
 
+**Loadbalancer**. To tell Zulip it's behind a loadbalancer, you can set
```
Comment (timabbott) [inline README.md:212]: This should be spelled "Load balancer".

**Comment by ben221199** [comment]
Fixed the issues.

File: README.md [hunk 4]
```diff
@@ -209,6 +209,13 @@ putting it in `/opt/docker/zulip/zulip/certs/` (by default, the
 `zulip` container startup script will generate a self-signed certificate and
 install it in that directory).
 
+**Load balancer**. To tell Zulip it's behind a loadbalancer, you can set
```
Comment (andersk) [inline README.md:212]: There are still two more instances of “loadbalancer” to be fixed to “load balancer”.

File: entrypoint.sh [hunk 5]
```diff
@@ -133,16 +133,24 @@ nginxConfiguration() {
...
     if [ "$QUEUE_WORKERS_MULTIPROCESS" == "True" ] || [ "$QUEUE_WORKERS_MULTIPROCESS" == "true" ]; then
         echo "Setting queue workers to run in multiprocess mode ..."
         crudini --set /etc/zulip/zulip.conf application_server queue_workers_multiprocess true
     elif [ "$QUEUE_WORKERS_MULTIPROCESS" == "False" ] || [ "$QUEUE_WORKERS_MULTIPROCESS" == "false" ]; then
         echo "Setting queue workers to run in multithreaded mode ..."
         crudini --set /etc/zulip/zulip.conf application_server queue_workers_multiprocess false
     else
-        echo "No additional puppet configuration executed."
-        return 0
+        echo "No additional puppet configuration executed for queue workers."
```
Comment (andersk) [inline entrypoint.sh:144]: The original code skipped the `zulip-puppet-apply -f` if there are no changes to be made. That should probably be preserved (e.g. using a flag variable).

**Comment by ben221199** [comment]
Fixed that issues too.

**Comment by alexmv** [comment]
Thanks, merged as e2503cd4c909ac1d969e559caad282aa421183fc!  I fixed the whitespace issues, and amended the wording on the commit message slightly.

I believe we're planning on a full 5.3 release early next week, so this will go out as part of that.

**Pull request closed without merging**